src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/ConnectionTest.php

```diff
@@ -326,16 +326,16 @@ public function testItDelaysTheMessage()
         $delayExchange->expects($this->once())->method('declareExchange');
         $delayExchange->method('getName')->willReturn('delay');
 
-        $delayQueue->expects($this->once())->method('setName')->with('delay_queue_5000');
+        $delayQueue->expects($this->once())->method('setName')->with('delay_queue__5000');
```

Comment:
The pattern is `delay_queue_%routing_key%_%delay%`. So when there's no routing key, you get the double `__`.